prow/github/types.go

```diff
@@ -1328,3 +1325,11 @@ type AppInstallationToken struct {
 	Permissions  InstallationPermissions `json:"permissions,omitempty"`
 	Repositories []Repo                  `json:"repositories,omitempty"`
 }
+
+// DirectoryContent contains information about a github directory.
+type DirectoryContent struct {
```

Comment:
Here too, can we link to the piece of doc that shows what GH response does this represent?